tests_metricflow/integration/test_cases/itest_metrics.yaml

```diff
@@ -551,6 +551,21 @@ integration_test:
         ds
     ) a
 ---
+integration_test:
```

Comment:
Maybe add one more test here for querying 1a and 1b together? Just to make sure that SQL also executes as expected

Reply:
Done!